gcc/jit/jit-recording.cc

```diff
@@ -4210,6 +4223,40 @@ recording::function::new_block (const char *name)
 void
 recording::function::write_to_dump (dump &d)
 {
+  for (auto attr: m_attributes)
```

Comment:
Did you do the write reproducer thing?

Reply:
More information please. :)

Reply:
Example: https://github.com/antoyo/gcc/blob/master/gcc/jit/jit-recording.cc#L3042-L3049